src/group_theory/perm/cycle_type.lean

```diff
@@ -145,6 +147,7 @@ cycle_induction_on (λ τ : perm α, τ.cycle_type.sum = τ.support.card) σ
   (λ σ hσ, by rw [hσ.cycle_type, coe_sum, list.sum_singleton])
   (λ σ τ hστ hc hσ hτ, by rw [hστ.cycle_type, sum_add, hσ, hτ, hστ.card_support_mul])
 
+/-- Compute signature from cycle type -/
```

Comment:
It seems very strange to me to claim that a lemma computes something

Reply:
:s/Compute/Express/